src/main/java/de/tum/in/www1/artemis/domain/Rating.java

```diff
@@ -0,0 +1,71 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * A Rating.
+ */
+@Entity
+@Table(name = "result_rating")
+@JsonInclude(Include.NON_EMPTY)
+public class Rating implements Serializable {
+
+    @Id
+    private Long id;
+
+    @Column(name = "rating")
+    private Integer rating;
+
+    @MapsId
+    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JoinColumn(name = "id")
+    private Result result;
```

Comment:
Depending on the usage a `@JsonIgnore` could be added since right now the whole `result` is always serialized when a rating is returned by the server although it's not actually needed for the current use case as far as I can see.